independent-publisher-connectors/Atlassian Jira/apiProperties.json

```diff
@@ -0,0 +1,45 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "email",
+          "description": "The email for this api",
+          "tooltip": "Provide the email",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide the API Key",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0052CC",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Set JIRA Address",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('yourInstance').atlassian.net/rest/api/3/"
```

Comment:
This connection parameter isn't defined. Add a new parameter called `jirainstance` and replace `yourinstance`.
```
    "connectionParameters": {
      "jirainstance": {
        "type": "string",
        "uiDefinition": {
          "displayName": "JIRA instance",
          "description": "The url where your JIRA instance is hosted (must support https). ",
          "tooltip": "Provide your JIRA domain (eg. myjirainstance.atlassian.net or mycompany.com:443/jira)",
          "constraints": {
            "required": "true"
          }
        }
      },
      "username": {
```